contrib/index.html

```diff
@@ -0,0 +1,199 @@
+  <!DOCTYPE html>
+  <html lang="en">
+    <head>
+      <meta charset="utf-8">
+      <meta http-equiv="X-UA-Compatible" content="IE=edge,chrome=1">
+      <meta name="viewport" content="width=device-width,initial-scale=1">
+      <title>Have fun ♥️</title>
```

Comment:
Is there an ASCII something we can use to make that more apparent in the code? It's just a rectangle here on Github for instance